app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -112,7 +112,7 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
           address={foundAddress?.address || ''}
         />
       )}
-      <BackButton includeBorder onClick={toPreviousStep} />
+      <BackButton role="link" includeBorder onClick={toPreviousStep} />
```

Comment:
Are we planning to prevent the spacebar from activating these fake links, since a link wouldn't be expected to be activated when pressing <kbd>Space</kbd>?

Reply:
Ideally no, and I don't think this alters the functionality of the HTML element, rather just how it is presented to AT. And I could've sworn spacebar still triggered the back button, but maybe I need to test again and in another browser.

Reply:
Ok, so I've tested the functionality of the back buttons in both Safari and Firefox and they respond to both Space and Enter/mouse click events.

Reply:
My expectation is that it wouldn't be activated when hitting <kbd>Spacebar</kbd>. It's a small detail, and maybe it ends up being helpful to have both options to activate, but if the idea with this is to make a link behave strictly as a link ought to behave, then I'd expect part of that to restrict interaction to the supported keypresses.

In general, I'm become a bit wary of this proposed behavior, but I see the upstream https://github.com/uswds/uswds/pull/4385 has recently been approved, so that train may have already left the station.

Reply:
I mean, your point is taken. What I'm doing here isn't really material honesty either, it's a button that is masquerading as a link, but still works like a button. At least now a screen reader will match what a sighted user might assume from the element's styling.